Sources/Workspace/Workspace.swift

```diff
@@ -1309,6 +1309,7 @@ extension Workspace {
     /// This will load the manifests for the root package as well as all the
     /// current dependencies from the working checkouts.
     // @testable internal
+    // FIXME: this logic should be changed to use identity instead of location once identity is unique
```

Comment:
this is the next step in the identity cleanup journey, but before its unique we need to keep using locations :(